### ISSUE TITLE
Question about normlize

### ISSUE THREAD

**Issue by qilong-zhang** [dongyp13/Non-Targeted-Adversarial-Attacks#15]
Hi, 
Why do you make img in [-1, 1], rather than (img - mean) / std?
I'm a little confused, could you help me?
Thanks!

**Comment by dongyp13** [comment]
Inception models require input images within [-1,1].

**Comment by qilong-zhang** [comment]
thanks! I get it

**Comment by Magallan1229** [comment]
But why pytorch implementation of inc v3 need images normalized using mean and std? Does " Inception models require input images within [-1,1]." only for tensorflow pretrained model?
See https://pytorch.org/hub/pytorch_vision_inception_v3/
